crates/solver/src/driver.rs

```diff
@@ -504,7 +504,7 @@ impl Driver {
                     Some(receipt.transaction_hash)
                 }
                 Err(SubmissionError::Revert(hash)) => Some(hash),
-                _ => None,
+                Err(other) => tracing::warn!("settlement submission error {err:?}"),
```

Comment:
Should still return `None` :smile`

Reply:
Also:
```rust
Err(err) => tracing::warn!(?err, "settlement submission error")
```

Reply:
Fixed. Wrote this without running check.